spring-cloud-dataflow-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/BaseDocumentation.java

```diff
@@ -37,10 +35,6 @@
  */
 public abstract class BaseDocumentation {
 
-	@ClassRule
```

Comment:
Should add this back since this code and the `@Before` gets duplicated too many times.  For the one test that needs a different class rule, there should be different inheritance hierarchy or no inheritance at all.